src/operator/account/AccountViewHeader.tsx

```diff
@@ -61,6 +63,16 @@ const AccountViewHeader: FC = () => {
           Convert to Contract
         </ButtonBase>
       )}
+      {hasWritePermissions && account?.reactivatable && (
+        <ButtonBase
+          color={ComponentColor.Primary}
+          shape={ButtonShape.Default}
+          onClick={() => setReactivateOverlayVisible(!reactivateOverlayVisible)}
```

Comment:
I know this is the convention elsewhere in this file - but should we be using `!reactivateOverlayVisible`? I think this is a one-way trip so really hitting the button should always set `reactivateOverlayVisible` to true.

Reply:
Yep! I definitely prefer that!